### PR TITLE
Update bugreport.yml

### PR DESCRIPTION
**Pull request by drobbins329**
Added a text box to include Discord.Net and related packages and versions, to assist in debugging issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 1]
```diff
@@ -76,3 +76,11 @@ body:
         ```
     validations:
       required: false
+  - type: textarea
```
Comment (csmir) [inline .github/ISSUE_TEMPLATE/bugreport.yml:79]: Partially duplicate of the 'Version' area. Would you care to rework this to only relate to additional packages? Discord.NET does not need to be specified.

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 2]
```diff
@@ -76,3 +76,11 @@ body:
...
+    id: packages
+    attributes: 
+      label: Packages
+      description: List Discord.Net and Discord.Net related pacakges in use, and versions.
```
Comment (csmir) [inline .github/ISSUE_TEMPLATE/bugreport.yml]: packages*

**Review by csmir — CHANGES_REQUESTED** [review]

**Comment by drobbins329** [comment]
fixed in latest commit

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 3]
```diff
@@ -80,7 +80,7 @@ body:
     id: packages
     attributes: 
       label: Packages
-      description: List Discord.Net and Discord.Net related pacakges in use, and versions.
-      placeholder: Discord.Net V3.3.2, etc.
+      description: List related packages in use, and versions.
```
Comment (csmir) [inline .github/ISSUE_TEMPLATE/bugreport.yml]: 'Please list all 3rd party packages in use if applicable, including their versions.' Sounds better to me, and resolves listing of packages that do not relate to Discord.NET.

**Review by csmir — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bugreport.yml (hunk 3 above)
Comment (drobbins329) [reply .github/ISSUE_TEMPLATE/bugreport.yml]: Agreed. That does sound better. Changed.

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 4]
```diff
@@ -80,7 +80,7 @@ body:
     id: packages
     attributes: 
       label: Packages
-      description: List related packages in use, and versions.
+      description: Please list all 3rd party packages in use if applicable, including their versions.
```
Comment (csmir) [inline .github/ISSUE_TEMPLATE/bugreport.yml:83]: Looks good to me 👍🏼

**Review by csmir — APPROVED** [review]

**Pull request merged**